hack/deployer/runner/ocp.go

```diff
@@ -182,7 +182,6 @@ func (d *OcpDriver) Execute() error {
 }
 
 func (d *OcpDriver) auth() error {
-	_ = NewCommand(fmt.Sprintf("gcloud config set project %s", d.ctx["GCloudProject"])).Run()
```

Comment:
Curious why you moved this to the bottom of the function.

Reply:
The json cred file already contains the region, so we don't want to temper the existing gcloud config on the host if we're using the JSON creds file.